cilium/cmd/bpf_egress_lookup.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2021 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"net"
+	"os"
+
+	"github.com/cilium/cilium/pkg/bpf"
+	"github.com/cilium/cilium/pkg/common"
+	"github.com/cilium/cilium/pkg/maps/egressmap"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	egressLookupUsage = "Lookup egress entries using source and destination IPs.\n"
+)
+
+var bpfEgressLookupCmd = &cobra.Command{
+	Use:   "lookup",
+	Short: "Lookup egress entries",
+	Long:  egressListUsage,
+	Run: func(cmd *cobra.Command, args []string) {
+		common.RequireRootPrivilege("cilium bpf egress update <src_ip> <dest_ip>")
+
+		if len(args) < 2 {
```

Comment:
You can use `Args: cobra.ExactlyNArgs(2),` in the declaration of `bpfEgressLookupCmd` instead of manually checking `len(args)`.

This comment applies to other commands in this PR too.